pyproject.toml

```diff
@@ -94,6 +94,7 @@ pytest-env = "1.1.3"
 pytest-randomly = "3.15.0"
 pytest-xdist = "3.6.1"
 safety = "3.2.0"
+trufflehog = "2.2.1"
```

Comment:
This is not valid, Trufflehog is a golang package, the Python version is the old version.

Reply:
Closing the PR then.